src/settings/Settings.js

```diff
@@ -152,10 +152,10 @@ export const SETTINGS = {
         default: null,
     },
     "feature_cross_signing": {
-        isFeature: true,
+        //isFeature: true,
         displayName: _td("Enable cross-signing to verify per-user instead of per-session (in development)"),
         supportedLevels: LEVELS_FEATURE,
-        default: false,
+        default: true,
```

Comment:
I know we've been down this road before, but I can't remember the answer, sorry... Why is it not enough to change just this one line here? In limited testing, it does seem to do the right thing, and it's still possible to override from a config file if desired.